client/apps/Admin/src/pages/AssignmentsPage.tsx

```
@@ -0,0 +1,453 @@
+
+        return Array.from(groups.values());
+    }, [assignments]);
+
+    const [editingAssignmentsTaskId, setEditingAssignmentsTaskId] = useState<string | null>(null);
+    const [deleteConfirmationTaskId, setDeleteConfirmationTaskId] = useState<string | null>(null);
+    const [isDeleting, setIsDeleting] = useState(false);
+
+    // Icons
+    const EditIcon = () => (
+        <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 24 24" fill="none" stroke="currentColor" strokeWidth="2" strokeLinecap="round" strokeLinejoin="round">
+            <path d="M17 3a2.828 2.828 0 1 1 4 4L7.5 20.5 2 22l1.5-5.5L17 3z" />
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)